<ui version="4.0" >
 <class>listitem</class>
 <widget class="QWidget" name="listitem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>123</width>
    <height>57</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QLabel" name="iconLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../resources/qpalringo.qrc" >:/contacts/pcContactIcon.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="firstLine" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>firstLine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="secondLine" >
       <property name="text" >
        <string>secondLine</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/qpalringo.qrc" />
 </resources>
 <connections/>
</ui>
